src/state/api/apiSlice.ts

```diff
@@ -258,6 +258,15 @@ export const apiSlice = createApi({
         return response
       },
     }),
+    getSkruAvKalkluatorFeatureToggle: builder.query<UnleashToggle, void>({
+      query: () => '/feature/pensjonskalkulator.skru-av-kalkulator',
```

Comment:
Togglen skal hete "pensjonskalkulator.vedlikeholdsmodus", ref [Slack 2025-03-14](https://nav-it.slack.com/archives/C04KM38NFS5/p1741951341591489?thread_ts=1741784487.109829&cid=C04KM38NFS5).

Man bør unngå at noe skrus AV når togglen skrus PÅ, derfor er "vedlikeholdsmodus" sånn sett å foretrekke.